cirq-core/cirq/contrib/acquaintance/devices.py

```diff
@@ -42,7 +42,7 @@ def validate_operation(self, operation: 'cirq.Operation') -> None:
 
 
 def is_acquaintance_strategy(circuit: 'cirq.Circuit') -> bool:
-    return isinstance(circuit._device, AcquaintanceDevice)
+    return True
```

Comment:
This seems weird and misleaeding that is_acquaintance_strategy is just always True.  Should we deprecate this function?

Reply:
It's not a part of the public API so nobody can call into this easily, but I'll go ahead and remove it.